packages/api/db/migration/20230106112659_add_extra_columns_irrigation_table.js

```diff
@@ -0,0 +1,43 @@
+export const up = async function (knex) {
+  await knex.schema.alterTable('irrigation_task', (table) => {
+    table.boolean('default_location_flow_rate').defaultTo(false);
+    table.boolean('default_location_application_depth').defaultTo(false);
+    table.boolean('default_irrigation_task_type_location').defaultTo(false);
+    table.boolean('default_irrigation_task_type_measurement').defaultTo(false);
+  });
+  const irrigation_task_type = await knex('irrigation_type').select(
+    'irrigation_type_id',
+    'irrigation_type_name',
+  );
+  await knex.schema.alterTable('irrigation_type', (table) => {
+    table.string('irrigation_type_translation_key');
+  });
+
+  for (const row of irrigation_task_type) {
+    const irrigation_type_translation_key = row.irrigation_type_name
+      .toUpperCase()
+      .split(' ')
+      .join('_');
+    await knex('irrigation_type')
+      .update({ irrigation_type_translation_key })
+      .where({ irrigation_type_id: row.irrigation_type_id });
+  }
+
+  await knex.schema.alterTable('location_defaults', (table) => {
```

Comment:
@wasswarichard 

The opposite of this should be in 'down' function.

Reply:
@Duncan-Brain this has been resolved